tests/UI/campaigns/functional/FO/08_menuAndNavigation/03_navigationAndDisplay/01_displayTags.ts

```diff
@@ -0,0 +1,442 @@
+    it('should go back BO', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goBackToBO1', baseContext);
+
+      await productPage.goTo(page, global.BO.URL);
+
+      const pageTitle = await dashboardPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(dashboardPage.pageTitle);
+    });
+
+    it('should go to \'Catalog > Products\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductsPage2', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.catalogParentLink,
+        dashboardPage.productsLink,
+      );
+      await productsPage.closeSfToolBar(page);
+
+      const pageTitle = await productsPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(productsPage.pageTitle);
+    });
+
+    it('should reset all filters', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'resetFilters', baseContext);
+
+      await productsPage.resetFilter(page);
+
+      const numberOfProducts = await productsPage.resetAndGetNumberOfLines(page);
+      await expect(numberOfProducts).to.be.above(0);
+    });
+
+    it('should filter by product name', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterProductName', baseContext);
+
+      await productsPage.filterProducts(page, 'name', Products.demo_6.name, 'input');
+
+      const numberOfProducts = await productsPage.getNumberOfProductsFromList(page);
+      await expect(numberOfProducts).to.eq(1);
+    });
+
+    it('should go to the product page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductPage2', baseContext);
+
+      await productsPage.goToEditProductPage(page, 1);
+
+      const pageTitle = await addProductPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(addProductPage.pageTitle);
+    });
+
+    it('should add a specific price', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'addSpecificPrice', baseContext);
+
+      const message = await addProductPage.addSpecificPrices(page, specificPriceData.specificPrice);
+      await expect(message).to.equal(addProductPage.settingUpdatedMessage);
+    });
+  });
+
+  describe('FO - Check the discount tag', async () => {
+    it('should preview product', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToShopFO2', baseContext);
+
+      page = await addProductPage.previewProduct(page);
+
+      const pageTitle = await productPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(Products.demo_6.name);
+    });
+
+    it('should check the discount tag', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkDiscountTag', baseContext);
+
+      const flagText = await productPage.getProductTag(page);
+      await expect(flagText).to.eq('-€10.00');
```

Comment:
Not possible to use a value from the product object ?

Reply:
Yes possible ;) Done